pkgs/by-name/ex/expo-orbit/package.nix

```
@@ -0,0 +1,114 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  dpkg,
+  alsa-lib,
+  cairo,
+  cups,
+  dbus,
+  expat,
+  libdrm,
+  glib,
+  pango,
+  nspr,
+  nss,
+  gtk3,
+  libgbm,
+  xorg,
+  autoPatchelfHook,
+  systemd,
+  makeWrapper,
+}: let
+  deps = [
+    alsa-lib
+    cairo
+    cups
+    dbus
+    expat
+    glib
+    gtk3
+    libdrm
+    libgbm
+    nspr
+    nss
+    pango
+    systemd
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libxcb
+  ];
+
+  version = "2.0.4";
+
+  selectSystem = attrs:
+    attrs.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
```

The `throw` is unnecessary as evaluation would fail far earlier than this on unsupported systems
```suggestion
    attrs.${stdenv.hostPlatform.system};
```